frontend/dockerfile/dockerfile_test.go

```diff
@@ -6518,7 +6518,7 @@ FROM scratch
 COPY --from=0 / /
 `)
 
-	const expectedDigest = "sha256:9e36395384d073e711102b13bd0ba4b779ef6afbaf5cadeb77fe77dba8967d1f"
+	const expectedDigest = "sha256:23bfe9c494f4b4ae9368d989035c70b3a34aa0bfc991618b3e54dcce2eee4bf8"
```

Comment:
Why did this change?

Reply:
The order of JSON field marshalling changes :cry: 

Essentially, the `mediaType` doesn't come first anymore (since it's not defined at the top-level).

Reply:
Could you give an example. Is the problem that `Version` comes before `Mediatype` now? FYI @AkihiroSuda

Reply:
Previously:

```json
{
  "mediaType": "application/vnd.oci.image.index.v1+json",
  "schemaVersion": 2,
  "manifests": [
    {
      "mediaType": "application/vnd.oci.image.manifest.v1+json",
      "digest": "sha256:27594184c4ad11392de489e7e9eba36e4685177b3df91a7b1e13e9dbec97d427",
      "size": 1056,
      "platform": {
        "architecture": "amd64",
        "os": "linux"
      }
    },
    {
      "mediaType": "application/vnd.oci.image.manifest.v1+json",
      "digest": "sha256:5a9a7a10ff48bf889374e0bb96dfd0f817fb96b0d86d9ff738543ecf2836c3bf",
      "size": 566,
      "annotations": {
        "vnd.docker.reference.digest": "sha256:27594184c4ad11392de489e7e9eba36e4685177b3df91a7b1e13e9dbec97d427",
        "vnd.docker.reference.type": "attestation-manifest"
      },
      "platform": {
        "architecture": "unknown",
        "os": "unknown"
      }
    }
  ]
}
```

Now:
```json
{
  "schemaVersion": 2,
  "mediaType": "application/vnd.oci.image.index.v1+json",
  "manifests": [
    {
      "mediaType": "application/vnd.oci.image.manifest.v1+json",
      "digest": "sha256:eb0454aabfa3084e80359c6dd64b2e88f26ca542a2af8a36eae752168bee1317",
      "size": 1056,
      "platform": {
        "architecture": "amd64",
        "os": "linux"
      }
    },
    {
      "mediaType": "application/vnd.oci.image.manifest.v1+json",
      "digest": "sha256:113c2a57ab2a69ec43f1fab69464c613697ae288cb24126422fc309939805311",
      "size": 566,
      "annotations": {
        "vnd.docker.reference.digest": "sha256:eb0454aabfa3084e80359c6dd64b2e88f26ca542a2af8a36eae752168bee1317",
        "vnd.docker.reference.type": "attestation-manifest"
      },
      "platform": {
        "architecture": "unknown",
        "os": "unknown"
      }
    }
  ]
}
```

`schemaVersion` appears first now (I think because marshaling visits each struct recursively, starting with the top-most, then descending down into embedded structs afterwards)

Reply:
Ah, yeah, noticed that as well some time ago; originally the MediaType part of specs.Versioned;
https://github.com/opencontainers/image-spec/commit/4267a18d918c4d87bb7423b9f1d6ae72b9ea9e76 (https://github.com/opencontainers/image-spec/pull/172)

Then got removed in https://github.com/opencontainers/image-spec/commit/5fc84e5f9d4b0a15bf293d44afd9266fededba64 (https://github.com/opencontainers/image-spec/pull/411)

And when it was added back, it was added to the manifest itself (instead of `Versioned`) https://github.com/opencontainers/image-spec/commit/5b82148fa647734a2c6095b2eab9b502fbca227e ()